mix.exs

```diff
@@ -1,41 +1,48 @@
 defmodule Safetensors.MixProject do
   use Mix.Project
 
-  @description "Safetensors dump and load in Elixir"
+  @version "0.1.1"
```

Comment:
@grzuy I don't think we need 0.2.0, 0.1.0 only has load and we are not breaking it, right?

Reply:
Hi @jonatanklosko,

Either works for me.
My interpretation from semantic versioning has always been anything 0.x could be potentially be considered breaking, so when updating 0.x branches I tend to update minor (nor tiny) digit.

Anyway, I published 0.2.0 20 minutes ago :-)
You guys good or prefer we retire?

Reply:
Oh, I changed `decode/1` to return iodata and not binary, so it is a breaking change, we would need 0.3 right away. You can unpublish the package version within 60 minutes, so I would just do that :)

Reply:
```
Reverted safetensors 0.2.0
```

Reply:
Awesome!